doc/user/tutorials/install.html

```
@@ -421,6 +442,26 @@ <h3>Building Crux From Source on MacOS</h3>
+		Ninja is an alternative build system which can be used instead of the 
+		default Makefiles,
+		and which can be much faster than Makefiles.
+		To use Ninja for building, first ensure that you have installed the <code>ninja</code> build system.
+		Then, configure the build with the following command:
+		<div class=code><code>
+		cmake -G Ninja -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX:PATH=&lt;install directory&gt; .
+		</code>
+		</div>
+		<p/>
+		You can then build and install using:
+		<div class=code><code>
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </div></li> ], start tag match failed [ <div class=code> ] on line 461.

(tag-pair)